CDP4Composition/Services/ValidationService.cs

```diff
@@ -78,7 +78,7 @@ public static class ValidationService
             { "Description", new ValidationRule { PropertyName = "Description", Rule = @"^(?!\s*$).+", ErrorText = "The Description must not be empty." } },
             { "Title", new ValidationRule { PropertyName = "Title", Rule = @"^(?!\s*$).+", ErrorText = "The Title must not be empty." } },
             { "EnumerationValueDefinitionShortName", new ValidationRule { PropertyName = "ShortName", Rule = @"^([^()\s][\S]*)$", ErrorText = "The ShortName can not be empty or contain a whitespace." } },
-            { "EnumerationValueDefinitionName", new ValidationRule { PropertyName = "Name", Rule = @"^([^()\s][\S]*)$", ErrorText = "The Name can not be empty or contain a whitespace." } }
+            { "EnumerationValueDefinitionName", new ValidationRule { PropertyName = "Name", Rule = @"^(?!\s*$).+", ErrorText = "The Name can not be empty." } }
```

Comment:
I think that in this case, starting the word with white space should also not be allowed.
So the correct one would be: @"^([^()\s][\S\s]*)$"

Also, I think we want to have this change in the Version 10 branch of Comet, which is branch "**feat/netoffice-net6**".